src/import_whatsapp.js

```diff
@@ -158,7 +158,7 @@ function processText(text) {
 	// stopping when new line starts with date or text ends
 	// Capture group 1 = date, group 2 = time, group 3 = sender, group 4 = message content
 	const messageRegex =
-		/(\d{2}\/\d{2}\/\d{4}),?\s(\d{1,2}:\d{2})(?:\s?(?:AM|PM|am|pm))?\s-\s(.*?):\s((.|\n)*?)(?=(\n\d{2}\/\d{2}\/\d{4})|$)/g;
+		/(\d{2}\/\d{2}\/\d{4}),?\s(\d{1,2}:\d{2})(?:\s?(?:AM|PM|am|pm))?\s-\s(.*?):[\t\f\cK ]((.|\n)*?)(?=(\n\d{2}\/\d{2}\/\d{4})|$)/g;
```

Comment:
what's the cK?

Reply:
after some digging it seems to be for vertical tab, which doesn't seem to be used much nowadays fwics? We could change to/include `\cJ` since this would look for `\n`

Reply:
The square brackets are supposed to match any whitespace except newline characters. I'm 99% sure we'll only ever encounter a space character in practice, but included all whitespace types just in case.